mars/contrib/dask/tests/test_dask.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from mars import new_session, stop_server
+from mars.contrib.dask import convert_dask_collection, mars_scheduler
+
+try:
+    import dask
+except ImportError:
+    dask = None
+
+try:
+    import mimesis
+except ImportError:
+    mimesis = None
+
+
+def setup_function():
+    new_session()
+
+
+def teardown_function():
+    stop_server()
+
+
+def require_dask_installed(func):
+    return pytest.mark.skipif(dask is None, reason='dask not installed')(func)
+
+
+def require_mimesis_installed(func):
+    return pytest.mark.skipif(mimesis is None, reason='mimesis not installed')(func)
+
+
+@require_dask_installed
+def test_delayed():
```

Comment:
How about just using fixture `setup_cluster`?

Reply:
I used `lazy_import` to keep it consistent with the other test scripts, is that ok?

Reply:
I used [`lazy_import` way](https://github.com/mars-project/mars/pull/2289/commits/908f1cef0700b231e1e92068d7e2ccaa14778ad5) to make it consistent with the other test scripts, is that ok?

Reply:
I mean the fixture: https://github.com/mars-project/mars/blob/322c187842a0ca99cea2a5f311d10e681969bd78/mars/conftest.py#L134

`setup_cluster` will be called only once across the entire CI, it will significantly save the time.

For the usage, just use it as the argument of a test function.

```python
def test_delayed(setup_cluster):
```

No import is required, because pytest knows how to handle it.

Hance, if `setup_cluster` is used, `setup_function` and `teardown_function` can be removed.

Reply:
Not sure if I get this, are you talking about [this way](https://github.com/mars-project/mars/pull/2289/commits/717b213d260c046feb77f96c604acf9a42b22e6f)?

Reply:
Exactly.